src/vcSettingsUI.cpp

```diff
@@ -219,7 +219,12 @@ void vcSettingsUI_Show(vcState *pProgramState)
             ImGui::Unindent();
           }
 
-          ImGui::SliderFloat(vcString::Get("settingsAppearanceSaturation"), &pProgramState->settings.presentation.saturation, 0.0f, 5.0f);
+          // limit the value between 0-5.
+          float saturation = pProgramState->settings.presentation.saturation;
+          ImGui::SliderFloat(vcString::Get("settingsAppearanceSaturation"), &saturation, 0.0f, 5.0f);
```

Comment:
This will work fine. But it could be improved slightly (remove the temporary variable, and use `udClamp()`).

```
// limit the value between 0-5.
if (ImGui::SliderFloat(vcString::Get("settingsAppearanceSaturation"), &pProgramState->settings.presentation.saturation, 0.0f, 5.0f))
  pProgramState->settings.presentation.saturation = udClamp(pProgramState->settings.presentation.saturation, 0.0f, 5.0f);
```

Reply:
Fixed.